cmd/talosctl/cmd/mgmt/cluster/create.go

```diff
@@ -436,6 +458,61 @@ func parseCPUShare() (int64, error) {
 	return nano.Num().Int64(), nil
 }
 
+func getDisks() ([]*provision.Disk, error) {
+	// should have at least a single primary disk
+	disks := []*provision.Disk{
+		{
+			Size: int64(clusterDiskSize) * 1024 * 1024,
+		},
+	}
+
+	for _, disk := range clusterDisks {
+		var (
+			partitions     = strings.Split(disk, ",")
+			diskPartitions = make([]*v1alpha1.DiskPartition, len(partitions))
+			diskSize       int64
+		)
+
+		for j, partition := range partitions {
+			parts := strings.Split(partition, ":")
+			if len(parts) != 2 {
+				return nil, fmt.Errorf("failed to parse malformed partition definition %s", partition)
+			}
+
+			partitionPath := parts[0]
+
+			if !strings.HasPrefix(partitionPath, "/") {
+				return nil, fmt.Errorf("partition path should be absolute")
```

Comment:
Good check. Something for another PR, but I wonder if this would be a good validation check in the config validator.

Reply:
yeah, we could validate in config in fact that it starts with `/var/` ?

Reply:
^ that too... assuming we don't think we have a use case for anything outside of var (_maybe_ `/etc/kubernetes`), and we would have to verify that no one is using it outside of `/var` already.

Reply:
great idea. Updated.